### ISSUE TITLE
Safe to use?

### ISSUE THREAD

**Issue by dgriego** [sixFingers/streamex#12]
given that this hasn't been updated in a quite a while, is it safe to assume this is no longer a community supported package for getstream?

**Comment by ronaldcurtis** [comment]
+1

**Comment by sixFingers** [comment]
That's right, I can no longer devote time to this package — if you find it useful, feel free to fork and I'll be glad to link directly to your project page.
